build-common.properties

```diff
@@ -62,6 +62,7 @@ hamcrest.lib=${framework.loc}/tests/hamcrest-core-1.3.jar
 
 checker.loc=${checkerframework}/checker
 checker.lib=${checker.loc}/dist/checker.jar
+checker.lib.tmp=${checker.loc}/dist/checker-tmp.jar
```

Comment:
Can you provide a comment what this file is for? The name implies it is temporary, but I don't see when it is removed.